### ISSUE TITLE
UCpt subdevice without prefix

### ISSUE THREAD

**Issue by ZryletTC** [pcdshub/pcdsdevices#455]
Not sure how hard or feasible this is to implement but it'd be nice to have because sometimes my classes are simply collections of signals without a proper prefix. Lemme know what you think @ZLLentz. Also, definitely not the highest priority so only take a look at this when you have time.

## Expected Behavior
If the subdevice doesn't expect a prefix, collect prefixes doesn't look for one.

## Current Behavior
It does anyway.

## Possible Solution
Some form of checking the signature?
Like I said, I'm not really sure if this is actually feasible without too much work.

## Context
Neither the FlowIntegrator nor the IntegratedFlow class in #420 should require prefixes but using UCpt currently forces the subdevice to use them.

**Comment by ZLLentz** [comment]
You're saying that these devices only contain `UCpt`, and have no direct signals of their own, so their prefix argument is not used in any meaningful way? Therefore, it is annoying to need to provide one?

**Comment by ZLLentz** [comment]
If so, then the plan is to add checks somewhere around https://github.com/pcdshub/pcdsdevices/blob/30c341da7c5ce7f59825716c90009139cf5adcd6/pcdsdevices/component.py#L71-L75

But we need to keep in mind that devices are not explicitly required to have their `prefix` argument be called `prefix`. It might also be parts of `*args`. But maybe it is ok to enforce conventions within the narrow band of "devices that are usable with `UCpt`"

**Comment by ZryletTC** [comment]
> You're saying that these devices only contain `UCpt`, and have no direct signals of their own, so their prefix argument is not used in any meaningful way? Therefore, it is annoying to need to provide one since `collect_prefixes` always assumes that a prefix argument exists?

Exactly. I haven't run into any situations where the prefix was stuffed into `*args`. I'd think it's probably ok to enforce  some conventions.
